application-ideas-ui/src/main/resources/Ideas/IdeasHomeSheet.xml

```diff
@@ -44,7 +44,7 @@ $xwiki.ssx.use("Ideas.IdeasClass")
 ## Load the User Picker style for the "Assigned to" column
 #set($discard = $xwiki.ssfx.use('uicomponents/widgets/userpicker/userPicker.css'))
 #set ($columnsProperties = {
-  'doc.fullName': {'type': 'text'},
+  'doc.fullName': {'type': 'text', 'headerClass' : 'hidden'},
```

Comment:
Why not ``'type': 'hidden'``?

Reply:
Using `'type' : 'hidden'` would not display the first column properly. 
![image](https://user-images.githubusercontent.com/45433221/146944270-106b757e-9810-4f0b-bf0e-af2073d788e5.png)